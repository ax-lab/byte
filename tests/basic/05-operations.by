print 1
print 'hello'
print true
print false
print (null)

print +false, +1, -2, -(-(+'3')), -+-+-+4
print not true, not not false, !0

print 1 + 4 * 10 * (2 * 6 % 10 + 2) / 2 / 2 + 1

print true ? 'true is true' : 'this is wrong'
print false ? 'this is wrong' : 'false is false' 
