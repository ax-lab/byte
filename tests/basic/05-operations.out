1
hello
true
false
null
0 1 -2 3 -4
false false 1
true is true
false is false
